lib/compute/agent-node-config.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+import {
+  AmazonLinuxCpuType, AmazonLinuxGeneration, InstanceClass, InstanceSize, InstanceType, MachineImage,
+} from '@aws-cdk/aws-ec2';
+import { Stack } from '@aws-cdk/core';
+import { AgentNodeConfig } from './agent-nodes';
+
+export class CloudAgentNodeConfig {
+  readonly AL2_X64: AgentNodeConfig;
+
+  readonly AL2_ARM64: AgentNodeConfig;
+
+  constructor(stack: Stack) {
+    this.AL2_X64 = {
+      ec2CloudName: 'AL2-X64',
+      instanceType: InstanceType.of(InstanceClass.C5, InstanceSize.XLARGE4).toString(),
+      workerLabelString: 'AL2-X64',
+      numberOfExecutors: '2',
+      remoteUser: 'ec2-user',
```

Comment:
Any reason we can't hardcode this value?

Reply:
I am using same groovy script to install 2 types of agent nodes inorder to avoid duplication. Maybe we can refactor later when we implement #44 . So we can have multiple groovy scripts for each type of agent node